packages/form-utils/package.json

```diff
@@ -44,6 +45,7 @@
     "typescript": "^4.6.4"
   },
   "peerDependencies": {
+    "@availity/mui-button": "workspace:*",
```

Comment:
I think we should list the `@availity/*` scoped packages as dependencies instead of dev/peer.

Reply:
I thought we had the mui components as peer/dev so we wouldn't run in to any issues when they're all dependencies of element.